docs/docs/t-digest.md

```diff
@@ -62,62 +62,56 @@ You measure the IP packets transferred over your network each second and try to
 
 ## Examples
 
-#### Creating a t-digest
+In the following example, you'll create a t-digest with a compression of 100 and add items to it. The `COMPRESSION` argument is used to specify the tradeoff between accuracy and memory consumption. The default value is 100. Higher values mean more accuracy. Note: unlike some of the other probabilistic data structures, the `TDIGEST.ADD` command will not create a new structure if the key does not exist.
 
-```
-> TDIGEST.CREATE my-tdigest COMPRESSION 100
-```
+{{< clients-example tdigest_tutorial tdig_start >}}
+> TDIGEST.CREATE bikes:sales COMPRESSION 100
+OK
+> TDIGEST.ADD bikes:sales 21
+OK
+> TDIGEST.ADD bikes:sales 150 95 75 34
+OK
+{{< /clients-example >}}
 
-The `COMPRESSION` argument is used to specify the tradeoff between accuracy and memory consumption. The default is 100. Higher values mean more accuracy.
-
-#### Adding a single element to the t-digest:
-```
-> TDIGEST.ADD my-tdigest 20.9
-```
-
-#### Adding multiple elements to the t-digest:
-```
-> TDIGEST.ADD my-tdigest 308 315.9
-```
 
 You can repeat calling [TDIGEST.ADD](https://redis.io/commands/tdigest.add/) whenever new observations are available
 
 #### Estimating fractions or ranks by values
 
 Another helpful feature in t-digest is CDF (definition of rank) which gives us the fraction of observations smaller or equal to a certain value. This command is very useful to answer questions like "*What's the percentage of observations with a value lower or equal to X*".
 
->More precisely, `TDIGEST.CDF` will return the estimated fraction of observations in the sketch that are smaller than X plus half the number of observations that are equal to X
-
-Let's illustrate this with an example: if we have a set of observations of people's age with gaussian distribution, we can ask a question like "What's the percentage of people younger than 50 years?"
+>More precisely, `TDIGEST.CDF` will return the estimated fraction of observations in the sketch that are smaller than X plus half the number of observations that are equal to X. We can also use the `TDIGEST.RANK` command, which is very similar. Instead of returning a fraction, it returns the ----estimated---- rank of a value. The `TDIGEST.RANK` command is also variadic, meaning you can send one number to query or multiple.
 
-```
-> TDIGEST.ADD my-tdigest 45.88 44.2 58.03 19.76 39.84 69.28 50.97 25.41 19.27 85.71 42.63
+Here's an example. If you have a set of observations of people's ages, you can ask a question like "What's the percentage of bike racers that are younger than 50 years?"
```

Comment:
Consider changing "if you have a set of observations of people's ages" to "Given a set of biker's ages"